chamge/types.c

```diff
@@ -0,0 +1,15 @@
+/**
+ *  Copyright 2019 SK Telecom, Co., Ltd.
+ *    Author: Jeongseok Kim <jeongseok.kim@sk.com>
```

Comment:
That's not me for this file. :)

And please split this commit into two;  *add backend error quark*, and *edge: make amqp error checking better*.

The latter depends on the former, but their topics are different.